superset/assets/javascripts/explorev2/components/ExploreViewContainer.jsx

```diff
@@ -61,7 +62,8 @@ class ExploreViewContainer extends React.Component {
     this.props.actions.removeControlPanelAlert();
     this.props.actions.removeChartAlert();
 
-    this.runQuery();
+    this.props.actions.triggerQuery();
```

Comment:
on this is `onQuery`, it just looks like it's `componentWillUnmount` because of the way Github folded the code...